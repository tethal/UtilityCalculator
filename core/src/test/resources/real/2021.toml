[general]
start_date = 2021-01-01
end_date = 2022-04-30
tenant = ["Marie Černá", "Jindřišská 16", "111 50 Praha 1"]
owner = ["Jan Novák", "majitel@example.com"]
report_place = "V Praze"
report_date = 2022-05-22
sources = ["Vyúčtování služeb od Společenství vlastníků za rok 2021", "Příloha č. 3 k vyhlášce č.269/2015 Sb.", "Měsíční předpis záloh pro rok 2022 od Společenství vlastníků", "https://www.pvk.cz/vse-o-vode/cena-vodneho-a-stocneho/vyvoj-vodneho-a-stocneho-v-praze/",]

[[deposits]]
description="celkem"
count=1
amount=43500

[[heating]]
annual_cost = 8712.9
start_date = 2021-01-01
end_date = 2021-12-31

[[heating]]
annual_cost = 8472
start_date = 2022-01-01
end_date = 2022-12-31

[[other_fees]]
start_date = 2021-01-01
end_date = 2021-12-31
annual_cost = 8772

[[other_fees]]
start_date = 2022-01-01
end_date = 2022-12-31
annual_cost = 8508

[[cold_water.reading]]
meter_id = "1"
reading_date = 2021-01-01
state = 158.1

[[cold_water.reading]]
meter_id = "1"
reading_date = 2021-10-15
state = 184.4

[[cold_water.reading]]
meter_id = "1"
reading_date = 2021-10-15
state = 0

[[cold_water.reading]]
meter_id = "1"
reading_date = 2022-01-01
state = 8.6

[[cold_water.reading]]
meter_id = "1"
reading_date = 2022-05-01
state = 21


[[cold_water.tariff]]
start_date = 2021-01-01
end_date = 2021-12-31
unit_amount = 103.4076

[[cold_water.tariff]]
start_date = 2022-01-01
end_date = 2022-12-31
unit_amount = 108.13

[[hot_water.reading]]
meter_id = "1"
reading_date = 2021-01-01
state = 96.6

[[hot_water.reading]]
meter_id = "1"
reading_date = 2021-10-15
state = 106.5

[[hot_water.reading]]
meter_id = "1"
reading_date = 2021-10-15
state = 0

[[hot_water.reading]]
meter_id = "1"
reading_date = 2022-01-01
state = 5.5

[[hot_water.reading]]
meter_id = "1"
reading_date = 2022-05-01
state = 13.3

[[hot_water.tariff]]
start_date = 2021-01-01
end_date = 2021-12-31
unit_amount = 81.495360

[[hot_water.tariff]]
start_date = 2022-01-01
end_date = 2022-12-31
unit_amount = 108.13

[[hot_water.heating_basic_cost]]
start_date = 2021-01-01
end_date = 2022-12-31
annual_cost = 1769.52

[[hot_water.heating_consumable_tariff]]
start_date = 2021-01-01
end_date = 2022-12-31
unit_amount = 150.998035
