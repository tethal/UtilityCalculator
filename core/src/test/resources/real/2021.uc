2021-01-01...2022-04-30
V Praze, 2022-05-22

[najemnik]
    Marie Černá
    Jindřišská 16
    111 50 Praha 1

[najemce]
    Jan Novák
    majitel@example.com

[zdroje]
    Vyúčtování služeb od Společenství vlastníků za rok 2021
    Příloha č. 3 k vyhlášce č.269/2015 Sb.
    Měsíční předpis záloh pro rok 2022 od Společenství vlastníků
    https://www.pvk.cz/vse-o-vode/cena-vodneho-a-stocneho/vyvoj-vodneho-a-stocneho-v-praze/


[zalohy]
    celkem: 1 x 43500

[vytapeni]
    2021: 8712.9
    2022: 8472

[ostatni poplatky]
    2021: 8772
    2022: 8508

[studena voda]
    O 2021-01-01: 158.1
    O 2021-10-15: 184.4
    O 2021-10-15: 0
    O 2022-01-01: 8.6
    O 2022-05-01: 21
    SV 2021: 103.4076
    SV 2022: 108.13

[tepla voda]
    O 2021-01-01: 96.6
    O 2021-10-15: 106.5
    O 2021-10-15: 0
    O 2022-01-01: 5.5
    O 2022-05-01: 13.3
    SV 2021: 81.49536
    SV 2022: 108.13
    ZS 2021...2022: 1769.52
    SS 2021...2022: 150.998035
