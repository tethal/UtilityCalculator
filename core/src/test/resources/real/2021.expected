Report:
  dateRange: 2021-01-01...2022-05-01
  tenant: Marie Černá, Jindřišská 16, 111 50 Praha 1
  owner: Jan Novák, majitel@example.com
  placeDate: V Praze, 2022-05-22
  sources: Vyúčtování služeb od Společenství vlastníků za rok 2021, Příloha č. 3 k vyhlášce č.269/2015 Sb., Měsíční předpis záloh pro rok 2022 od Společenství vlastníků, https://www.pvk.cz/vse-o-vode/cena-vodneho-a-stocneho/vyvoj-vodneho-a-stocneho-v-praze/
  sections:
    Studená voda: 4949.74
    Teplá voda: 7960.94
    Vytápění: 13626.66
    Ostatní poplatky: 11608.00
    Přijaté zálohy: -43500
    TOTAL: -5354.66

Studená voda:
  readings:
    [1] 2021-01-01...2021-10-15: 158.1...184.4 = 26.3
    [1] 2021-10-15...2022-01-01: 0...8.6 = 8.6
    [1] 2022-01-01...2022-05-01: 8.6...21 = 12.4
  priceList:
    2021-01-01...2021-10-15: 26.300 * 103.4076 = 2719.62
    2021-10-15...2022-01-01: 8.600 * 103.4076 = 889.31
    2022-01-01...2022-05-01: 12.400 * 108.13 = 1340.81

Teplá voda:
  readings:
    [1] 2021-01-01...2021-10-15: 96.6...106.5 = 9.9
    [1] 2021-10-15...2022-01-01: 0...5.5 = 5.5
    [1] 2022-01-01...2022-05-01: 5.5...13.3 = 7.8
  priceList:
    2021-01-01...2021-10-15: 9.900 * 81.49536 = 806.80
    2021-10-15...2022-01-01: 5.500 * 81.49536 = 448.22
    2022-01-01...2022-05-01: 7.800 * 108.13 = 843.41
  heatingBasicParts:
    2021-01-01...2022-05-01: 16.00 * 147.46 = 2359.36
  heatingConsumableParts:
    2021-01-01...2021-10-15: 9.900 * 150.998035 = 1494.88
    2021-10-15...2022-01-01: 5.500 * 150.998035 = 830.49
    2022-01-01...2022-05-01: 7.800 * 150.998035 = 1177.78

Vytápění:
  2021-01: 1.00 * 8712.9 * 0.19 = 1655.45
  2021-02: 1.00 * 8712.9 * 0.16 = 1394.06
  2021-03: 1.00 * 8712.9 * 0.14 = 1219.81
  2021-04: 1.00 * 8712.9 * 0.09 = 784.16
  2021-05: 1.00 * 8712.9 * 0.02 = 174.26
  2021-06: 1.00 * 8712.9 * 0.0 = 0.00
  2021-07: 1.00 * 8712.9 * 0.0 = 0.00
  2021-08: 1.00 * 8712.9 * 0.0 = 0.00
  2021-09: 1.00 * 8712.9 * 0.01 = 87.13
  2021-10: 1.00 * 8712.9 * 0.08 = 697.03
  2021-11: 1.00 * 8712.9 * 0.14 = 1219.81
  2021-12: 1.00 * 8712.9 * 0.17 = 1481.19
  2022-01: 1.00 * 8472 * 0.19 = 1609.68
  2022-02: 1.00 * 8472 * 0.16 = 1355.52
  2022-03: 1.00 * 8472 * 0.14 = 1186.08
  2022-04: 1.00 * 8472 * 0.09 = 762.48

Ostatní poplatky:
  2021-01-01...2022-01-01: 12.00 * 731.00 = 8772.00
  2022-01-01...2022-05-01: 4.00 * 709.00 = 2836.00

Přijaté zálohy:
  celkem: 1 * 43500 = 43500
